.github/workflows/cherry-pick.yml

```diff
@@ -26,6 +26,16 @@ jobs:
           fetch-depth: 0
           token: ${{ secrets.AMPPROJECTBOT }}
 
+      - name: Generate cherry-pick branch name
+        run: |
+          AMP_VERSION=${{ github.event.inputs.amp-version }}
+          read -r -a SHAS_ARRAY <<< "${{ github.event.inputs.shas }}"
+          SHAS_COUNT=${#SHAS_ARRAY[@]}
+          CURRENT_CP_COUNT=${AMP_VERSION:10:13}
+          NEW_CP_COUNT=00$(expr $CURRENT_CP_COUNT + $SHAS_COUNT)
+          CHERRY_PICK_BRANCH=amp-release-${AMP_VERSION:0:10}${NEW_CP_COUNT: -3}
```

Comment:
Took me a while to figure out how this ensures it's always 3 digits even if the number goes above 009... the fact that the whitespace before the `-3` is important is just... _cringe in bash_ 😱

I dunno about readability, I feel like at this point we should just install Node after the CP, create an `amp` task called `amp get-version` that just prints the return value of `getVersion` from `build-system/compile/internal-version.js`, and call it a day

wdyt?

Reply:
Using `internal-version.js` is more involved because you have to checkout the tag and it's hard to test.

This is what I was trying to avoid by naming the branches `cherry-pick-*`.

I'd rather keep the ugly bash and call this a day.

Reply:
Fair enough! Maybe add a couple of comments to clarify the less-than-ideal lines here, but otherwise I'm okay with yoloing this